gpflow/transforms.py

```diff
@@ -126,7 +126,7 @@ class Log1pe(Transform):
     A transform of the form
     .. math::
 
-       y = \log(1 + \exp(x))
+       y = \log(x)
```

Comment:
Why this change? This transform is softplus, not log

Reply:
I needed the log transform for my HMC sampler and I didn't know at that time how to change the file.